### ISSUE TITLE
README is missing the definition of the utility provided by this project

### ISSUE THREAD

**Issue by mikeputnam** [dougabugg/python-factorio-utility#1]
I like Python.
I like Factorio.
I like the idea of programmatic utility.
What are the big picture goals of this?

**Comment by dougabugg** [comment]
Good point, I should add more content to the README! At first, I just wanted this script to help me build more optimized factories and supply chains, but I envision in the future, (maybe, if I ever have time 😛 ) building a "bot" that can automatically play factorio, and eventually optimize the bot and compete for the fastest ["tool-assisted-speedrun" of factorio](https://www.speedrun.com/Factorio)! Or something crazy like that... I am bad at managing my time and finishing projects lol
